cmd/openfish/entities/annotations.go

```diff
@@ -36,9 +36,13 @@ package entities
 
 import (
 	"encoding/json"
+	"fmt"
+	"strings"
 
 	"github.com/ausocean/openfish/cmd/openfish/types/timespan"
 	"github.com/ausocean/openfish/datastore"
+
+	googlestore "cloud.google.com/go/datastore"
```

Comment:
I haven't looked at why we need this but generally we shouldn't be using the google datastore package since it breaks our ausocean datastore abstraction layer.

Reply:
Yes, I agree.  There should be no direct calls to any cloud.google.com/go/datastore from any of our application code.

We've gone to great effort to create an abstraction layer and it has brought us many benefits, especially when testing. Therefore, if there is additional functionality that we deem essential, it should be implemented in openfish/datastore.